.github/workflows/main.yml

```diff
@@ -43,4 +45,21 @@ jobs:
         run: mix deps.compile
 
       - name: Run tests
-        run: mix test.integration --trace
+        run: mix test --trace
+
+      - name: Install protoc
+        if: ${{matrix.integration}}
+        run: |
+          wget https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
+          unzip protoc-3.12.3-linux-x86_64.zip
+          echo "./bin" >> $GITHUB_PATH
+
+      - name: Compile .proto files to Elixir with protoc
+        if: ${{matrix.integration}}
```

Comment:
I don't know if we should actually flag it like this or move it to a separate `job`.

Reply:
I think this is great 👍